judo-tatami-psm2asm/src/main/epsilon/transformations/asm/psmToAsm.etl

```diff
@@ -19,5 +19,5 @@ post {
         }
     }
 
-    asmUtils.validateUniqueXmiids();
+    // asmUtils.validateUniqueXmiids();
```

Comment:
I suggest creating a ticket for validating unique xmiid-s only if it's required by setting a build parameter.

Reply:
nvm